Source/Matter/CCZ4RHSWithMatter.hpp

```diff
@@ -87,19 +87,22 @@ class MatterCCZ4RHS : public CCZ4RHS<gauge_t, deriv_t>
        It allows the user to set the value of Newton's constant, which is set to
        one by default.
     */
-    MatterCCZ4RHS(matter_t a_matter, params_t a_params, double a_dx,
-                  double a_sigma, int a_formulation = CCZ4RHS<>::USE_CCZ4,
-                  double a_G_Newton = 1.0);
+    CCZ4RHSWithMatter(params_t a_params, double a_dx, double a_sigma,
+                      int a_formulation = CCZ4RHS<>::USE_CCZ4,
+                      double a_G_Newton = 1.0);
 
     //!  The compute member which calculates the RHS at each point in the box
     //!  \sa matter_rhs_equation()
-    template <class data_t> void compute(Cell<data_t> current_cell) const;
+    template <class data_t>
+    AMREX_GPU_DEVICE AMREX_FORCE_INLINE void
+    compute(int i, int j, int k, const amrex::Array4<data_t> &rhs,
+            const amrex::Array4<data_t const> &state) const;
```

Comment:
Let's make these consistent with the definition.
```suggestion
    compute(int i, int j, int k, const amrex::Array4<data_t> &rhs_arrays,
            const amrex::Array4<data_t const> &state_arrays) const;
```